components/LaunchPage/HearFromHappyLaunchers.jsx

```diff
@@ -0,0 +1,102 @@
+import Image from 'next/image';
+
+import { SUB_HEADER_CLASS, TEXT_SMALL_CLASS } from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { ExternalLink } from 'common-util/ExternalLink';
+
+const TweetHeader = () => (
+  <div className="flex items-start gap-4 md:items-center">
+    <a
+      href="https://x.com/koeppelmann"
+      target="_blank"
+      rel="noreferrer"
+      className="rounded-full cursor-pointer"
+    >
+      <Image
+        src="/images/launch-page/testimony-avatar.png"
+        alt="avatar"
+        width={48}
+        height={48}
+      />
+    </a>
+
+    <div>
+      <h3 className="font-bold">
+        Martin Köppelmann, Co-Founder and CEO of Gnosis
+      </h3>
+      <ExternalLink
+        href="https://x.com/koeppelmann"
+        className={TEXT_SMALL_CLASS}
+      >
+        @koeppelmann
+      </ExternalLink>
+    </div>
+
+    <a
+      href="https://gnosis.io"
+      target="_blank"
+      rel="noreferrer"
+      className="ml-auto rounded-full cursor-pointer"
+    >
+      <Image
+        src="/images/launch-page/gnosis-logo.svg"
+        alt="gnosis"
+        width={48}
+        height={48}
+      />
+    </a>
+  </div>
+);
+
+const TweetBody = () => (
+  <p>
+    Another big user has been AI agents by
+    {' '}
+    <ExternalLink href="https://x.com/autonolas">@autonolas</ExternalLink>
+    {' - '}
+    we have this ongoing challenge where agents gather information and trade
+    information and trade on prediction markets - this has lead to the majority
```

Comment:
the text is a bit different from the original tweet, is it expected? the tweet says `... gather information and trade on prediction markets..` where we have `... gather information and trade information and trade..`